packages/js/src/ui/internal/parseMarkdown.tsx

```diff
@@ -0,0 +1,36 @@
+export interface Token {
```

Comment:
Is Token a standard terminology in the Markdown realm?

Reply:
Yes, it seems so. [ref](https://github.com/markedjs/marked/blob/master/src/Tokens.ts).